index.bs

```diff
@@ -2237,8 +2237,8 @@ partial dictionary MLOpSupportLimits {
   </tr>
   <tr>
     <td>*output*</td>
-    <td>{{MLArgMinMaxOptions/outputDataType}}</td>
-    <td>{{input}}'s [=MLOperand/rank=] - 1 to {{input}}'s [=MLOperand/rank=]</td>
+    <td>{{MLOperandDataType/"int32"}}, {{MLOperandDataType/"int64"}}</td>
```

Comment:
Note: This should list all allowed data types. The original outputDatatype is more useful for output data type calculation in the algorithm steps.